Blazorade.Bootstrap.Components/Blazorade.Bootstrap.Components.csproj

```diff
@@ -19,8 +19,9 @@
 
 
   <ItemGroup>
-    <PackageReference Include="Blazorade.Core" Version="0.9.0-preview" />
+    <PackageReference Include="Blazorade.Core" Version="0.9.0.2-preview" />
     <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Analyzers" Version="3.1.1" />
```

Comment:
I think all of these Microsoft.AspNetCore libraries go hand in hand with the version numbers, so I guess they all should be referencing the same ASP.NET Core version. Since ASP.NET Core 3.1 is not yet released, I would rather stick with v3.0 for now.